tiatoolbox/cli.py

```diff
@@ -434,5 +440,96 @@ def tissue_mask(
             )
 
 
+@main.command()
+@click.option(
+    "--predefined_model",
+    help="Predefined model used to process the data. the format is "
+    "<model_name>_<dataset_trained_on>. For example, `resnet18-kather100K` "
+    "is a resnet18 model trained on the kather dataset.",
+    default="resnet18-kather100K",
+)
+@click.option(
+    "--pretrained_weight",
+    help="Path to the model weight file. If not supplied, the default "
+    "pretrained weight will be used.",
+    default=None,
+)
+@click.option(
+    "--img_input",
```

Comment:
please use a name consistent with previous cli functions.

Reply:
There is not really a similar situation to compate to? Others are usually `wsi_input`

Reply:
Changed wsi input to img_input to keep consistent. Refer to [this commit](https://github.com/TIA-Lab/tiatoolbox/pull/98/commits/cdd19b7a3668e4c65e991dbff2d467faef64de05).